[2021-10-03 06:39:46,251] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: dag1.upload_file_to_hdfs_7 2021-10-03T06:39:28.166559+00:00 [queued]>
[2021-10-03 06:39:46,378] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: dag1.upload_file_to_hdfs_7 2021-10-03T06:39:28.166559+00:00 [queued]>
[2021-10-03 06:39:46,380] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-03 06:39:46,381] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-03 06:39:46,383] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-03 06:39:46,441] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): upload_file_to_hdfs_7> on 2021-10-03T06:39:28.166559+00:00
[2021-10-03 06:39:46,490] {standard_task_runner.py:52} INFO - Started process 1049 to run task
[2021-10-03 06:39:46,677] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'upload_file_to_hdfs_7', '2021-10-03T06:39:28.166559+00:00', '--job-id', '59', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/upload_file_to_staging.py', '--cfg-path', '/tmp/tmplvacfnn1', '--error-file', '/tmp/tmpyjk9x0xa']
[2021-10-03 06:39:46,696] {standard_task_runner.py:77} INFO - Job 59: Subtask upload_file_to_hdfs_7
[2021-10-03 06:39:48,115] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.upload_file_to_hdfs_7 2021-10-03T06:39:28.166559+00:00 [running]> on host 405455ff482b
[2021-10-03 06:39:49,769] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=mrk.jse@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=upload_file_to_hdfs_7
AIRFLOW_CTX_EXECUTION_DATE=2021-10-03T06:39:28.166559+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-03T06:39:28.166559+00:00
[2021-10-03 06:39:49,799] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-03 06:39:49,873] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo "Uploading transactions_7.csv..."']
[2021-10-03 06:39:50,430] {subprocess.py:74} INFO - Output:
[2021-10-03 06:39:51,258] {subprocess.py:78} INFO - Uploading transactions_7.csv...
[2021-10-03 06:39:51,277] {subprocess.py:82} INFO - Command exited with return code 0
[2021-10-03 06:39:51,490] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=upload_file_to_hdfs_7, execution_date=20211003T063928, start_date=20211003T063946, end_date=20211003T063951
[2021-10-03 06:39:51,887] {local_task_job.py:209} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-10-03 06:39:51,897] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 1049
[2021-10-03 06:39:51,925] {process_utils.py:66} INFO - Process psutil.Process(pid=1049, status='terminated', exitcode=0, started='06:39:46') (1049) terminated with exit code 0
